fh_immuta_utils/data_source.py

```diff
@@ -128,6 +96,22 @@ class DataSource(BaseModel):
     useDatesAsDirectory: bool = False
 
 
+class SchemaEvolutionMetadataConfigTemplate(BaseModel):
+    nameFormat: str
+    tableFormat: str
```

Comment:
Could we call this `sqlTableNameFormat` to be consistent with `sqlSchemaNameFormat` and variable names in code?
Or even queryEngineTableNameFormat and queryEngineSchemaNameFormat

Reply:
These are the exact key names that are passed to and consumed by the API when this object is converted to a dictionary.

Reply:
On second thought, I forgot that pydantic has aliases, so this should be doable. I'll look into it.

Reply:
Added those aliases as you suggested. They are now `dataSourceNameFormat`, `queryEngineTableNameFormat`, and `queryEngineSchemaNameFormat`